examples/multi-service-profile/main.tf

```diff
@@ -51,14 +51,15 @@ locals {
   # Merge zone ids to pass as contexts to the rule
   target_services_details = [
     {
-      target_service_name = "kms",
+      target_service_name = "secrets-manager",
```

Comment:
Made changes in target service from "kms" to "secrets-manager" in this example because in fscloud example same resource group (geretain-test-cbr) was used to create rule for `kms` as the pipeline tests were failing with `A rule with the same resource attributes already exists`.